### ISSUE TITLE
check in bulk

### ISSUE THREAD

**Issue by khmuhtadin** [AbhishekBiswal/whatsapp-verify#3]
dear developer, this is an awesome project.

is there any way to check bulk number by uploading csv file or something?

**Comment by bilalgul99** [comment]
I can not figure Out How to use the Check in Bulk Method, If some documentation could be included it would be really helpful

**Comment by AbhishekBiswal** [comment]
@khmuhtad1n No, there's no included method for bulk checking, but the same method (CLI / JS function) can be used to bulk check numbers - that would require some work.

@bilalgul99 There's no documentation for it - but I'll try to include one as soon as I can

**Comment by chat2alexio** [comment]
what is the cmd for bulk check

**Comment by sshah23** [comment]
Hello @AbhishekBiswal - could you provide further documentation on the Bulk Check please :)
